glue-codes/openfast/CMakeLists.txt

```diff
@@ -42,3 +42,6 @@ if(BUILD_OPENFAST_LIB_DRIVER)
 endif()
 
 
+add_library(dummyclib SHARED
```

Comment:
Is this needed for something?  I'm not sure I understand the use case.

Reply:
remove later: https://github.com/orgs/OpenFAST/projects/9/views/1?pane=issue&itemId=108648606